### PR TITLE
Examples were added to svdvals().

### LINKED ISSUE #1255: svdvals
Reformat svdvals #1258

### PR DESCRIPTION
**Pull request by karlschwarz**
This pull request fixes #1255

- Examples were added to svdvals() function in linear_algebra. 

Thanks for the review.

### CODE REVIEW

**Review by MarShaikh — CHANGES_REQUESTED** [review]
The examples seem fine but the formatting isn't right. Can you please follow the formatting [here](https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html) exactly? Thanks!

File: ivy/functional/ivy/linear_algebra.py [hunk 1]
```diff
@@ -612,6 +612,94 @@ def svdvals(
...
+    >>> print(x.shape)
+    (4, 3)
+    >>> S = ivy.svdvals(x)
+    >>> print(S)
+    ivy.array([10.3  ,  1.16 ,  0.615])
+
+    Compare the singular value S by ivy.svdvals() with the result by ivy.svd().
+    >>> _, SS, _ = ivy.svd(x)
+    >>> print(SS)
+    ivy.array([10.3  ,  1.16 ,  0.615])
```
Comment (MarShaikh) [inline ivy/functional/ivy/linear_algebra.py:647]: Can you please leave a single newline after each output of the print statement? It is needed as `ivy_tests/test_docstrings.py` is set up in such a way that it expects the output of a print statement to end at '\n'.

Comment (karlschwarz) [reply ivy/functional/ivy/linear_algebra.py:649]: @MarShaikh Sure, it is done! Thanks for your review and advice!

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Hi, I noticed that you were making changes to an older commit. Can you push these changes you made in a newer commit as otherwise you would be missing one of the other examples here. Sorry for being a little nitpicky!

**Comment by karlschwarz** [comment]
I am sorry I got a little confused, so how should I do that? It seems that the codes in my repository is the latest version and I cannot commit another newer commit without any changes to the code. Please let me know how can I do that, thanks!

**Comment by MarShaikh** [comment]
> @MarShaikh I am sorry I got a little confused, so how should I do that? It seems that the codes in my repository is the latest version and I cannot commit another newer commit without any changes to the code. Please let me know how can I do that, thanks!

Hey, sorry for the delayed response, I was trying to see if there was a way of fixing the issue. It seems like this is a known bug from Github's end. That is when you make a PR to the base branch from a feature branch and the base branch changes, github files shows the diff based on the particular commit when you made the PR. 

I was able to find one [solution](https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch/26986320#26986320). You can read [this](https://github.com/isaacs/github/issues/750) issue to read more about it.

**Comment by karlschwarz** [comment]
> > @MarShaikh I am sorry I got a little confused, so how should I do that? It seems that the codes in my repository is the latest version and I cannot commit another newer commit without any changes to the code. Please let me know how can I do that, thanks!
> 
> Hey, sorry for the delayed response, I was trying to see if there was a way of fixing the issue. It seems like this is a known bug from Github's end. That is when you make a PR to the base branch from a feature branch and the base branch changes, github files shows the diff based on the particular commit when you made the PR.
> 
> I was able to find one [solution](https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch/26986320#26986320). You can read [this](https://github.com/isaacs/github/issues/750) issue to read more about it.

Thanks for the information! I will study the solution and try to solve this issue.

**Comment by karlschwarz** [comment]
@MarShaikh Hi, I changed the base branch to the other branch (not master) as the first solution in the [solution](https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch/26986320#26986320). I plan to change the branch back to master but now my PR is closed. Could you help me to solve this problem? Thanks a lot!

**Comment by karlschwarz** [comment]
@MarShaikh I also made another PR #1743 which is the same as the #1258. I hope this might be helpful. Thanks for your work.

**Pull request closed without merging**